document/initialization.md

```diff
@@ -12,8 +12,19 @@
 - `$ wget https://dlcdn.apache.org/maven/maven-3/3.8.8/binaries/apache-maven-3.8.8-bin.tar.gz`
 - `$ tar -xvf apache-maven-3.8.8-bin.tar.gz`
 
+- install check `$ java -version`
+
 ### Install MySQL
 - `$ sudo dnf install mysql mysql-server`
+- `$ sudo systemctl enable mysqld` (for auto restart)
+- active check `$ sudo systemctl status mysqld`
```

Comment:
``` 
$ sudo systemctl status mysqld
● mysqld.service - MySQL 8.0 database server
     Loaded: loaded (/usr/lib/systemd/system/mysqld.service; enabled; preset: d>
     Active: active (running) since Sat 2024-07-20 21:27:19 KST; 9min ago
    Process: 621 ExecStartPre=/usr/libexec/mysql-check-socket (code=exited, sta>
    Process: 648 ExecStartPre=/usr/libexec/mysql-prepare-db-dir mysqld.service >
   Main PID: 711 (mysqld)
     Status: "Server is operational"
      Tasks: 37 (limit: 10704)
     Memory: 450.5M
        CPU: 3.138s
     CGroup: /system.slice/mysqld.service
             └─711 /usr/libexec/mysqld --basedir=/usr

Jul 20 21:27:17 ip-172-31-41-61.ap-southeast-2.compute.internal systemd[1]: Sta>
Jul 20 21:27:19 ip-172-31-41-61.ap-southeast-2.compute.internal systemd[1]: Sta>
lines 1-15/15 (END)
```